Source/WebCore/html/HTMLSummaryElement.cpp

```diff
@@ -96,12 +98,28 @@ bool HTMLSummaryElement::isActiveSummary() const
     return details->isActiveSummary(*this);
 }
 
-static bool isClickableControl(EventTarget* target)
+static bool isInInteractiveContentWithinSummary(EventTarget* target)
```

Comment:
`isInSummaryInteractiveContent` ?

Reply:
Thanks — made it `isInSummaryInteractiveContent`